entity-management-web/src/test/java/eu/europeana/entitymanagement/util/EntityDataReconceliationTest.java

```diff
@@ -0,0 +1,93 @@
+package eu.europeana.entitymanagement.util;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Test;
+
+import eu.europeana.entitymanagement.definitions.model.impl.BaseTimespan;
+import eu.europeana.entitymanagement.definitions.model.impl.BaseWebResource;
+import eu.europeana.entitymanagement.web.service.impl.MetisDereferenceService;
+
+/**
+ * JUnit tests to merge the data from a different entity (a so called entity proxy) into the given entity
+ */
+public class EntityDataReconceliationTest {
+
+	private static final Logger logger = LogManager.getLogger(MetisDereferenceService.class);
+	
+	@Test
+	public void mergeEntities() {
```

Comment:
an unit test for each entity Type needs to be created. The input data must be read from json or xml files.

Reply:
I leave this for now since it would require a bit more work. This is because you asked me to finish as soon as possible to merge your changes.